test-cases/performance/perf-regression-2mv.yaml

```diff
@@ -3,11 +3,12 @@ test_duration: 90
 
 stress_cmd_no_mv: "cassandra-stress user profile=/tmp/cs_no_mv_basic_profile.yaml ops'(insert=1)' cl=QUORUM duration=30m -pop seq=1..10000000 -mode cql3 native -rate threads=100"
 stress_cmd_no_mv_profile: 'data_dir/cs_no_mv_basic_profile.yaml'
-stress_cmd_mv:
```

Comment:
@roydahan the test was been used since before we move away from avocado configuration...

should we delete it ?